inz-demo/id-card/Issues/address/description.xml

```diff
@@ -0,0 +1,34 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo ID Card Address</en>
+        <nl>Demo ID Card Address</nl>
```

Comment:
```suggestion
        <nl></nl>
```

Reply:
We need to remove all content within `nl` tags everywhere on this PR.

Reply:
In the current credentials, I saw we only removed nl translations from the attributes! because of that, I kept these!

Reply:
Actually, you must have `ln` translation for ` <Name>` tags!